contrib/charts/cert-manager/issuers/clusterissuer.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: certmanager.k8s.io/v1alpha1
```

Comment:
missing the check here `{{- if .Values.clusterIssuer.email }}`

Reply:
this file is rendered by the configmap.
So this if is only needed there.

Reply:
Can we add a comment at the top of the file indicating this is rendered here, and then created out-of-band, to prevent that confusion?